frontend/dockerfile/docs/experimental.md

```diff
@@ -138,3 +138,21 @@ $ buildctl build --frontend=dockerfile.v0 --local context=. --local dockerfile=.
 You can also specify a path to `*.pem` file on the host directly instead of `$SSH_AUTH_SOCK`.
 However, pem files with passphrases are not supported.
 
+### RUN --security=insecure
```

Comment:
I'd prefer `--security=sandbox|insecure` for completeness. You can tweak the next sentence a bit so that it only applies if the value is set to "insecure".

Reply:
updated!